site/docs/mfa/webhooks/about.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: aboutMfaWebhooks
+title: MFA Webhooks
+slug: /mfa/webhooks
+description: About Bandwidth's MFA Webhooks
+keywords:
+  - mfa
+  - callbacks
+  - webhooks
+  - bandwidth
+hide_title: false
+sidebar_label: About
+custom_edit_url: github.com/Bandwidth/api-docs/tree/main/site/docs/mfa/webhooks/about.mdx
+---
+# Two-Factor Authentication Webhooks
+Two-Factor Authentication (2FA) Webhooks (also known as Callbacks) rely on DLRs (Delivery Receipts) sent to the Two-Factor Authentication (2FA) platform by both Bandwidth's North America and International platform. Since these two platforms send different DLR formats, the 2FA platform standardizes them into a singular format for simplicity and ease of consumption. The 2FA platform then takes these standardized DLRs and sends them via an HTTP POST to the specified URL that was set in the `callbackUrl` field in the [initial request](../../messaging/quickStart.mdx#configuring-callback-urls). The `callbackUrl` field can be any publicly addressable url.
```

Comment:
To make it consistent with (2FA), I say put DLRs in parenthesis instead of Delivery Receipts and change the order, so it would be "rely on Delivery Receipts (DLRs)...."

Also, the second (2FA) isn't necessary.

Reply:
Probably worth including messaging in the description of the platforms, since we don't send any webhooks for voice. Ex: "...by both Bandwidth's North America and International messaging platforms"

Reply:
Nit: no need to include "standardized" in the sentence after we describe the DLR format as standardized, "The 2FA platform then takes these DLRs...."

Reply:
Same with "specifed", "and sends them via an HTTP POST to the URL that was set in the..."

Reply:
updated